save-points/01-Components-and-layout/BlazingPizza.ComponentsLibrary/LocalStorage.cs

```diff
@@ -5,13 +5,13 @@ namespace BlazingPizza.ComponentsLibrary
 {
     public static class LocalStorage
     {
-        public static Task<T> GetAsync<T>(IJSRuntime jsRuntime, string key)
+        public static ValueTask<T> GetAsync<T>(IJSRuntime jsRuntime, string key)
             => jsRuntime.InvokeAsync<T>("blazorLocalStorage.get", key);
 
-        public static Task SetAsync(IJSRuntime jsRuntime, string key, object value)
+        public static ValueTask<object> SetAsync(IJSRuntime jsRuntime, string key, object value)
```

Comment:
should this use `InvokeVoidAsync`?

Reply:
Yes, looks like it could. Since you'll probably be tweaking this during your updates (since the `.js` file should no longer be needed), I'll leave it with you to decide.